### ISSUE TITLE
Better error message in case of invalid arg type annotated

### ISSUE THREAD

**Issue by xsveda** [sockeqwe/fragmentargs#11]
When I create a simple POJO and annotate it with `@Arg` it cannot be of course used as a fragment argument but the only info I get is that build fails with message `java.lang.IllegalStateException`.

Is it possible to tune this error message? Something like `java.lang.IllegalStateException: Arg "mUser" cannot be serialized to Bundle`?

**Comment by sockeqwe** [comment]
Hi, could you just post your sample, just to ensure that we are talking about the same issue.
```java
class Pojo {
   int a;
   String foo;
}


class MyFragment extends Fragment {
    @Arg Pojo pojo;
}

```
This is will cause the problem, since `Pojo` **does not** implement `Parcelable`, right?

**Comment by xsveda** [comment]
Yes, this is absolutely the situation I was refering to!

**Comment by sockeqwe** [comment]
ok, i will fix that with next version 2.1 (end of march)
